getpaid/utils.py

```diff
@@ -81,3 +82,16 @@ def qs_to_ordered_params(query_string):
             field = field.decode('utf8')
         params[field] = value
     return params
+
+
+def get_domain(request=None):
+    if request and request.META.get('HTTP_HOST'):
```

Comment:
think is that someone maybe don't want to use SITE_URL at all because it's used by other apps, it will be nice to have an opportunity to configure getpaid independently